.circleci/config.yml

```
@@ -945,7 +948,7 @@ jobs:
                   - steps: << parameters.before-steps >>
                 # Make both `test-files` and `source-folder` parameters optional. Translates to: if `parameter` => run associated flag (`--spec` and `--folder`, respectively)
                 command: |
-                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >>
+                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >> <<# parameters.currents-record >> --record << parameters.currents-record >> <</ parameters.currents-record >> <<# parameters.source-folder >> --group << parameters.source-folder >> <</ parameters.source-folder >>
                 after-steps:
```

I should've probably done this earlier, but now is a good time to keep logical commands in their own separate lines.

Also, there are two flags (`--folder` and `--group`) that depend on the same parameter (`parameters.source-folder`) so it makes sense to group them, rather than to repeat the same conditional logic in two places.
```suggestion
                  run test-cypress-no-build
                    <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >>
                    <<# parameters.source-folder >>
                      --folder << parameters.source-folder >>
                      --group << parameters.source-folder >>
                    <</ parameters.source-folder >>
                    <<# parameters.currents-record >> --record << parameters.currents-record >>
                after-steps:
```

NOTE: This is just a suggestion. It might not work if you commit it directly because of the whitespace.